### DISCUSSION TITLE
伊朗解除网络限制

### DISCUSSION THREAD

**Discussion by osongsongo** [XTLS/Xray-core#4215] [General]
伊朗把墙取消了，不知道这对代理社区来说是好事还是坏事，以后软件开发人员肯定会变少，~~不过可能规范上更加统一了？~~

**Comment by shuiliu11** [comment]
哪里的新闻？你也太高看伊朗了，代理本来就是因为墙国才起来的，伊朗只是分红。

**Reply by osongsongo** [reply]
https://www.zaobao.com.sg/realtime/world/story20241225-5646066

**Comment by RPRX** [comment]
伊朗逐步取消墙的话对伊朗人民来说是好事，~~其实我觉得对代理社区来说也是好事，没有明文 HTTP 面板和乱封 REALITY 了~~

但中国就别想了，偶尔看到群里有人讨论拆墙我都想笑，国内那么多出版物、影视剧审查等法律法规都依附于墙的存在，有半点可能拆吗？
